trino-connector/integration-test/src/test/java/org/apache/gravitino/trino/connector/integration/test/TrinoQueryRunner.java

```diff
@@ -65,6 +67,29 @@ private QueryRunner createQueryRunner() throws Exception {
   }
 
   String runQuery(String query) {
+    String retryFlag = "<RETRY WITH NOT EXISTS>";
```

Comment:
Using spaces here is prone to errors. For example, there might be two spaces or Chinese spaces. It's recommended to use "_" instead.

Reply:
Fixed.